patches/tModLoader/Terraria/Projectile.cs.patch

```diff
@@ -1495,6 +1495,24 @@
  		if (aiStyle == 1) {
  			AI_001();
  		}
+@@ -17290,6 +_,8 @@
+ 							continue;
+ 
+ 						if (type == 10) {
++							if (TileLoader.Convert(num96, num97, BiomeConversionID.PurificationPowder))
```

Comment:
Is this correctly placed? What about the `if (type == 11 || type == 463)` below?